packages/cli/test/integration/passwordReset.endpoints.test.ts

```diff
@@ -62,6 +62,7 @@ afterAll(async () => {
 
 test('POST /forgot-password should send password reset email', async () => {
 	const authlessAgent = utils.createAgent(app);
+	const member = await testDb.createUser({ email: 'test@test.com' });
```

Comment:
Or `randomEmail()`. This is changed by #3066.

Reply:
yes, but just wanted to make sure that the email is uppercaseable. `randomEmail()` also consists of numbers as far as I've seen so I chose to define it before the test might get flaky